src/modules/Core/Account/EdgeAccountCallbackManager.js

```diff
@@ -0,0 +1,83 @@
+// @flow
+
+import type { EdgeAccount } from 'edge-core-js'
+import React from 'react'
+import { connect } from 'react-redux'
+
+import { updateExchangeRates } from '../../ExchangeRates/action'
+import type { Dispatch, State } from '../../ReduxTypes.js'
+import { updateWalletsRequest } from '../Wallets/action.js'
+
+type EdgeAccountCallbackManagerStateProps = {
+  account: EdgeAccount
+}
+
+type EdgeAccountCallbackManagerDispatchProps = {
+  updateWalletsRequest: () => any,
+  updateExchangeRates: () => any
+}
+
+type Props = EdgeAccountCallbackManagerStateProps & EdgeAccountCallbackManagerDispatchProps
+
+class EdgeAccountCallbackManager extends React.Component<Props> {
```

Comment:
Should this extend `PureComponent`? If our parent re-renders, we might re-render as well, which could cause double subscribes.